src/PackagePrivate/ValueObjects/Composites/Date.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace EDTF\PackagePrivate\ValueObjects\Composites;
```

Comment:
I like what you did with this parser state stuff. Yay Value Objects. What about putting all of this in a "Parser" namespace, or something similar? (Inside PackagePrivate)